### DISCUSSION TITLE
Can I access /data/data with Shizuku API?

### DISCUSSION THREAD

**Discussion by CyanChanges** [RikkaApps/Shizuku#203] [Q&A]
I am creating a backup App.
Can I access /storage/emulated/0/Android/data or /data/data Folder with Shizuku API
If it's possible, please give the code. Thanks!

**Comment by LOLIMHUMAN** [comment]
Shizuku can't access /data/data without root AFAIK. You should be able to access the ext. data (Android/data) folder with no problem, at least prior to  Android SDK 30. You probably can do so even in newer versions, I read about a [workaround](https://stackoverflow.com/a/66466205) a while back, but it got fixed in Android 13. Don't ask me how to do so despite the patch, just use your Google-Fu. I'm totally clueless about it too :D
